docs/releases/2.0.txt

```diff
@@ -177,6 +177,9 @@ Management Commands
 * :djadmin:`inspectdb` now translates MySQL's unsigned integer columns to
   ``PositiveIntegerField`` or ``PositiveSmallIntegerField``.
 
+* :djadmin:`makemessages` now accepts the optional argument ``--add-location``
+  for ``gettext`` 0.19 or newer.
```

Comment:
I would add a sentence to briefly explain the purpose of this argument. Maybe something like:
```
:djadmin:`makemessages` now accepts the optional argument ``--add-location`` 
to control the comments format in PO files. This feature is only available when 
`gettext` 0.19 or newer is available.
```